### ISSUE TITLE
Allow to choose to download results

### ISSUE THREAD

**Issue by SylvainGirod** [pink-room/fastlane-plugin-run_tests_firebase_testlab#13]
In some cases, we may not want to download results from the bucket.

There could be an optional parameter allowing us to explicitly say that we do not want to get the raw results.

Example :
```
lane :test do
  run_tests_firebase_testlab(
    project_id: "your-firebase-project-id",
    model: "Nexus6P",
    version: "27",
    delete_firebase_files: false,
    download_results_from_firebase: false
  )
end
```

**Comment by bffcorreia** [comment]
Sounds good for me. I will implement it soon 🙂

**Comment by SylvainGirod** [comment]
As I got some time, I tried to develop something.
Hope it will help you !

**Comment by bffcorreia** [comment]
Thank you for your first contribution @SylvainGirod ! The PR is perfect. I will merge it and deploy a new version 🙂

**Comment by bffcorreia** [comment]
Implemented in version [0.2.2](https://rubygems.org/gems/fastlane-plugin-run_tests_firebase_testlab/versions/0.2.2)!
